### PR TITLE
fix(Japanese) : Update and modify Japanese translations

### PR DESCRIPTION
**Pull request by KoukiHama**
Signed-off-by: Kouki Hama <kouki1.hama@toshiba.co.jp>

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change?

Issue: 　#1336 

If you are not my colleague and you understand Japanese, please review it.

Or please check whether sw360 show Japanese or not.

### CODE REVIEW

**Comment by KoukiHama** [comment]
I just changed the language property  file, but the github action is failing to build....

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 1]
```diff
@@ -3,22 +3,22 @@ accept.request=アクセスリクエスト
...
-add=Add
+active=アクティブ
+active.status=アクティブステータス
+add=追加
 add.attachment=添付ファイルの追加
 add.comment=コメントの追加
 add.component=コンポーネントの追加
 add.data.to.this.release=このリリースにデータを追加
-add.license.info.to.release=Add License Info to Release
+add.license.info.to.release=ライセンス情報をリリースに追記
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 2]
```diff
@@ -93,11 +92,11 @@ AttachmentType-REQUIREMENT=要求事項記載ドキュメント
 AttachmentType-SCAN_RESULT_REPORT=ライセンス用スキャナが見つけたものを出力したレポート
 AttachmentType-SCAN_RESULT_REPORT_XML=ライセンススキャナが見つけたものをXMLで出力したもの
 AttachmentType-SCREENSHOT=スクリーンショット、通常はライセンス情報のあるウェブサイトのスクリーンショット
-AttachmentType-SOURCE=オリジナルコースコード
+AttachmentType-SOURCE=オリジナルソースコード
 AttachmentType-SOURCE_SELF=自作のソースコード配布
 attachment.usages=添付ファイルの用途
 attachment.usages2=添付ファイルの用途
-attachment.deletion.from.local.fs=Attachment Deletion From Local FS
+attachment.deletion.from.local.fs=ローカルファイルシステムにより削除された
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 3]
```diff
@@ -123,7 +122,7 @@ by.clicking.on.the.checkbox.in.the.header.row=ヘッダーのチェックボッ
 cancel=キャンセル
 cancel.all.scheduled.tasks=予定されたタスクをすべてキャンセルする
 cancel.cve.service=CVEサービスのキャンセル
-cancel.scheduled.attachment.deletion.from.local.fs=Cancel Scheduled Attachment Deletion From Local FS
+cancel.scheduled.attachment.deletion.from.local.fs=ローカルファイルシステムの定期的な削除をキャンセルする
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 4]
```diff
@@ -133,13 +132,13 @@ cannot.link.to.release=リリースへのリンクができません。
...
+categories=カテゴリ
+category=カテゴリ
 changed.attachments=変更済添付ファイル
 change.log=変更履歴
-change=Change
-rating=Rating
-change.rating.and.action.of.selected.vulnerabilities=Change rating and action of selected vulnerabilities
+change=変更
+rating=レート
+change.rating.and.action.of.selected.vulnerabilities=選択した脆弱性の評価の変更
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 5]
```diff
@@ -330,20 +328,20 @@ darcs=Darcs
 database.administration=データベース管理
 database.sanitation=データベース状態
 date=日付
-date.should.be.valid=Date should be valid
-days=Days
-deactivated=Deactivated
-deactivate.user=Deactivate User
+date.should.be.valid=データ適切であるべき
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 6]
```diff
@@ -1437,23 +1437,23 @@ you.do.not.have.write.access.to.the.project= <b>WRITE</b> 書き込み権限無
...
+license.obligation=ライセンスオブリゲーション
+obligation.title=オブリゲーションタイトル
+obligation.text=オブリゲーションテキスト
+linked.obligations=リンクされたオブリゲーション
+learn.more.about.obligation.level=よりオブリゲーション詳細について
+ObligationLevel=組織オブリゲーション。組織のオブリゲーションとは、ソフトウェアを伝達する前に確認すべき一般的な規則または必須の手順です。 &#10例えば，プロジェクト、ソフトウェア、または組織に質問がある場合に、OSS連絡先の電子メールアドレスを、伝達されるソフトウェアのすべてのインスタンスに追加することが挙げられます。 &#10コンポーネントオブリゲーション コンポーネントオブリゲーションは、特定のコンポーネントまたはリリースのみに対するオブリゲーションです。 &#10例えば、実行されるべき特別な措置またはアクションは、貿易コンプライアンスまたはコンポーネントのIP問題に起因する可能性があります。 &#10ライセンスオブリゲーション。ライセンスオブリゲーションとは、特定のライセンスの下でのソフトウェアの使用から実行されるべきタスクまたは考慮されるべきリスクのことです。 &#10プロジェクトオブリゲーション。プロジェクトのオブリゲーションは、プロジェクトや製品の性質に特有のもので、ソフトウェアを &#10提供する前に確認すべきステップやタスクを必要とします。 &#10例えば、印刷されたドキュメントが限られている小さなハードウェアが考えられます。この場合、オープンソースのライセンス情報には特別な対応が必要となり、例えば、OSSのライセンス情報を取得する方法をパッケージに印刷します。
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 7]
```diff
@@ -1437,23 +1437,23 @@ you.do.not.have.write.access.to.the.project= <b>WRITE</b> 書き込み権限無
...
+ObligationLevel-ORGANISATION_OBLIGATION=組織オブリゲーションとは、ソフトウェアを提供する前に確認すべき一般的なルールや必須の手順のことです。 例えば、伝達されるソフトウェアのすべてのインスタンスに、プロジェクト、ソフトウェア、または組織に質問がある場合は、OSSの連絡先の電子メールアドレスを追加することが挙げられます。
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 8]
```diff
@@ -1437,23 +1437,23 @@ you.do.not.have.write.access.to.the.project= <b>WRITE</b> 書き込み権限無
...
+ObligationLevel-COMPONENT_OBLIGATION=コンポーネントオブリゲーションとは、特定のコンポーネントまたはリリースのみに対する義務です。 &#10、例えば、実行されるべき特別な措置またはアクションは、コンポーネントの貿易コンプライアンス &#10、またはIP問題に起因するものです。
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 9]
```diff
@@ -1437,23 +1437,23 @@ you.do.not.have.write.access.to.the.project= <b>WRITE</b> 書き込み権限無
...
+ObligationLevel-LICENSE_OBLIGATION=ライセンスオブリゲーションとは、特定のライセンスの下で &#10ソフトウェアを使用する際に、実施すべきタスクや考慮すべきリスクのことです
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 10]
```diff
@@ -1437,23 +1437,23 @@ you.do.not.have.write.access.to.the.project= <b>WRITE</b> 書き込み権限無
...
+ObligationLevel-PROJECT_OBLIGATION=プロジェクトオブリゲーションは、プロジェクトや製品の性質に固有としてあるもので、ソフトウェアを提供する前に確認しておくべきステップやタスクを指します。この場合、オープンソースのライセンス情報を入手するための手順をパッケージに印刷するなど、特別な対応が必要となります。
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I fixed by your comments

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 11]
```diff
@@ -330,20 +328,20 @@ darcs=Darcs
 database.administration=データベース管理
 database.sanitation=データベース状態
 date=日付
-date.should.be.valid=Date should be valid
-days=Days
-deactivated=Deactivated
-deactivate.user=Deactivate User
+date.should.be.valid=データ適切であるべき
+days=日
+deactivated=アクティベートされてない
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 12]
```diff
@@ -330,20 +328,20 @@ darcs=Darcs
...
+deactivate.user=アクティベートされてないユーザ
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 13]
```diff
@@ -3,22 +3,22 @@ accept.request=アクセスリクエスト
 access=アクセス
 access.denied=アクセス拒否
 access.token=アクセストークン
-access.token.validity=Access Token Validity
+access.token.validity=アクセストークンの有効性
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 14]
```diff
@@ -3,22 +3,22 @@ accept.request=アクセスリクエスト
...
 accountant=会計士
 a.component.with.the.same.name.already.exists=同名のコンポーネントが既に存在します。
 action=アクション
 actions=アクション
 action.taken=実行されたアクション
-activate.user=Activate user
+activate.user=アクティブユーザ
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 15]
```diff
@@ -48,7 +47,7 @@ agreed.clearing.date.yyyy.mm.dd=クリアリング承認日 YYYY-MM-DD
 al=輸出管理
 alienbrain=Alienbrain
 all=すべての
-all.of.the.directly.linked.releases.are.cleared=All of the directly linked release(s) are cleared.
+all.of.the.directly.linked.releases.are.cleared=リンクされたリリースのクリアが全て完了
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 16]
```diff
@@ -393,13 +391,13 @@ download.license.archive=ライセンスアーカイブダウンロード
 download.liferay.users=Liferayユーザーダウンロード
 download.release.link.information=リリースリンク情報ダウンロード
 download.release.link.sample.information=リリースリンクサンプル情報ダウンロード
-do.you.really.want.to.activate.the.user.x=Do you really want to activate the user <b data-name="email"></b>
+do.you.really.want.to.activate.the.user.x=本当にアクティブユーザ削除しますか？ <b data-name="email"></b>
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 17]
```diff
@@ -393,13 +391,13 @@ download.license.archive=ライセンスアーカイブダウンロード
...
 do.you.really.want.to.clean.up.the.attachment.database=本当に添付ファイルのデータベースをクリーンアップしますか？
 do.you.really.want.to.add.license.info.to.all.the.directly.linked.releases=Do you really want to add license info to all the directly linked release(s)
-do.you.really.want.to.deactivate.the.user.x=Do you really want to deactivate the user <b data-name="email"></b>
+do.you.really.want.to.deactivate.the.user.x=本当にアクティブでないユーザを削除しますか？ <b data-name="email"></b>
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 18]
```diff
@@ -461,7 +459,7 @@ enable.e.mail.notifications=電子メール通知を有効にする
 enable.security.vulnerability.monitoring=セキュリティ脆弱性の監視を有効にする
 end.user=エンドユーザー
 enter.access.token=アクセストークンを入力
-enter.access.token.validity=Enter access token validity
+enter.access.token.validity=有効なアクセストークンを入力
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 19]
```diff
@@ -499,36 +497,35 @@ enter.owner.billing.group=会計グループオーナーを入力
 enter.owners.accounting.unit=所有者の会計単位を入力
 enter.owners.billing.group=会計グループオーナーを入力
 enter.project=プロジェクトを入力
-enter.refresh.token.validity=Enter Refresh Token Validity
+enter.refresh.token.validity=トークン有効期間入力
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 20]
```diff
@@ -539,43 +536,43 @@ error.fetching.license.to.source.file.mapping=Error fetching license to source f
 error.fetching.project.from.backend=バックエンドからプロジェクトを取得する際にエラーが発生
 error.fetching.release.from.backend=バックエンドからリリースを取得する際にエラーが発生
 error.in.looking.for.duplicate.identifiers=重複する識別子の検索でエラーが発生
-error.while.creating.user.record.it.could.be.new.user.email.or.global.id.is.used.by.another.user=Error while creating user record. It could be new user email or global id is used by another user
-error.while.activating.or.deactivating.user.record=Error while activating or deactivating user record.
-error.while.updating.user.information.it.could.be.new.user.email.or.global.id.is.used.by.another.user=Error while updating user information. It could be new user email or global id is used by another user
+error.while.creating.user.record.it.could.be.new.user.email.or.global.id.is.used.by.another.user=ユーザ作成エラー．新しいメールアドレスかIDが既に登録済かもしれません．
+error.while.activating.or.deactivating.user.record=ユーザ登録情報をアクティブあるいは非アクティブする際にエラー
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 21]
```diff
@@ -672,29 +670,29 @@ internal=内部
 internal.project=社内プロジェクト
 internal.use=内部利用
 interval=インターバル
-invalid.agreed.clearing.date=Invalid agreed clearing date!
-invalid.clearingTeam.email=Invalid clearingTeam email!
-invalid.clearing.request.id=Invalid clearing request id!
-invalid.preferred.clearing.date=Invalid preferred clearing date!
-invalid.requested.clearing.date=Invalid requested clearing date!
+invalid.agreed.clearing.date=承認済のクリアリングデータが不正
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 22]
```diff
@@ -672,29 +670,29 @@ internal=内部
...
+invalid.clearingTeam.email=クリアリングチームのメールアドレスが不正
+invalid.clearing.request.id=クリアリングリクエストIDが不正
+invalid.preferred.clearing.date=参照されているクリアリングデータが不正
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 23]
```diff
@@ -672,29 +670,29 @@ internal=内部
...
+invalid.requested.clearing.date=リクエストされたクリアリングデータが不正
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 24]
```diff
@@ -794,7 +793,7 @@ modified.on=変更された
 monotone=Monotone
 more.info=詳細情報
 more.than.1000.documents.affected.the.merge.operation.might.time.out.in.consequence.only.some.of=1000以上のドキュメントが影響を受けています。結果として、一部のマージ操作がタイムアウトする可能性があります。
-multiple.approved.cli.are.found.in.the.release=Multiple approved CLI are found in release
+multiple.approved.cli.are.found.in.the.release=複数承認されたCLIがリリースに見つかりました．
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 25]
```diff
@@ -842,11 +841,11 @@ no.releases.linked.to.this.project.or.its.linked.projects=このプロジェク
 no.report=レポートはありません
 no.results.found.please.refine.your.search=結果が見つかりませんでした。
 no.subscriptions.available=サブスクリプションはできません。
-none.of.the.directly.linked.releases.are.cleared=None of the directly linked release(s) are cleared.
+none.of.the.directly.linked.releases.are.cleared=クリアリング済のリンクされているリリース無し
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 26]
```diff
@@ -842,11 +841,11 @@ no.releases.linked.to.this.project.or.its.linked.projects=このプロジェク
...
 not.applicable=適用不可
 not.available=利用できない
 not.checked=未チェック
-in.analysis=In Analysis
+in.analysis=分析
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 27]
```diff
@@ -929,27 +929,29 @@ please.enter.an.url=URLを入力してください
...
-please.enter.preferred.clearing.date.to.reopen.clearing.request=Please enter preferred clearing date to re-open clearing request
+please.enter.preferred.clearing.date.to.reopen.clearing.request=クリアリングリクエストを再開するための希望クリアリング日を入力してください
 please.enter.the.clearing.team.email.id=クリアリングチームのメールIDを入力してください．
 please.enter.the.password=パスワードを入力してください
 please.enter.the.title=タイトルを入力してください
 please.enter.the.server.url=サーバーのURLを入力してください
 please.enter.the.user.name=ユーザー名を入力してください
 please.enter.user.key=ユーザーキーを入力してください
-please.enter.valid.access.token.validity=Please enter valid access token validity
+please.enter.valid.access.token.validity=適切なアクセストークンを入力してください．
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 28]
```diff
@@ -929,27 +929,29 @@ please.enter.an.url=URLを入力してください
...
+please.enter.valid.refresh.token.validity=適切リフレッシュトークンを丹生リクしてください．
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 29]
```diff
@@ -1026,7 +1027,7 @@ re.check.connection=接続の再確認
 references=参考
 reference.doc=参考ドキュメント
 refresh=リフレッシュ
-refresh.token.validity=Refresh Token Validity
+refresh.token.validity=有効なトークンのリフレッシュ
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 30]
```diff
@@ -1135,33 +1136,33 @@ search.licenses=ライセンスの検索
...
 search.vendor=検索ベンダー
-seconds=Seconds
-secondary.department=Secondary Department
-secondary.departments.and.roles=Secondary Departments and Roles
-secondary.departments.and.roles.edited.successfully.for.user=Secondary Departments and Roles edited successfully for user
-secondary.department.role=Secondary Department Role
+seconds=秒
+secondary.department=第2所属部門
+secondary.departments.and.roles=第2所属部門とそのロール
+secondary.departments.and.roles.edited.successfully.for.user=ユーザの第2所属部門の変更が成功
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 31]
```diff
@@ -1219,7 +1220,7 @@ subscribers=加入者
 subversion.(svn)=Subversion (SVN)
 SUCCESS=成功
 success.please.reload.page.to.see.the.changes=成功! 変更点を確認するには、ページをリロードしてください！
-suggested.linked.project.enable.svm=Suggested Linked Project Enable SVM
+suggested.linked.project.enable.svm=プロジェクトにリンク済SVM(Security Vulnerability Monitor)提案
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 32]
```diff
@@ -1327,25 +1327,25 @@ upload.license.archive=ライセンスアーカイブのアップロード
...
-UserGroup=User: &#10Admin: &#10Clearing Admin: &#10ECC Admin: &#10Security Admin: &#10SW360 Admin
-user.information=User Information
-user.preferences=ユーザー設定
-user.not.found.in.couch.db=User not found in Couch DB
-user.not.found.in.liferay.db=User not found in Liferay DB
-user.role=ユーザーロール
-users=ユーザー
-users.already.in.couchdb=Users already in Couch DB
-users.not.in.couchdb=Users not in Couch DB
+UserGroup=ユーザ: &#10Admin: &#10Clearing Admin: &#10ECC Admin: &#10Security Admin: &#10SW360 Admin
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 33]
```diff
@@ -1327,25 +1327,25 @@ upload.license.archive=ライセンスアーカイブのアップロード
...
+user.information=ユーザ情報
+user.preferences=ユーザ設定
+user.not.found.in.couch.db=CouchDBにいないユーザ
+user.not.found.in.liferay.db=LiferayDBにいないユーザ
+user.role=ユーザロール
+users=ユーザ
+users.already.in.couchdb=Couch DBに既にいるユーザ
+users.not.in.couchdb=Couch DBにいない
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 34]
```diff
@@ -1364,16 +1364,16 @@ version=バージョン
...
+view.file.list=ファイルリストビュー
 view.obligations=オブリゲーションを表示
 visibility=可視範囲
 Visibility=Private: 作成者（およびすべての表示レベルに適用される管理者）のみが表示できます。&#10Me and Moderators: 作成者とモデレーターが表示できます。&#10Group and Moderators: 同じグループメンバーとモデレータが見れる &#10Everyone: このシステムにログインしている人全員が見れる
 Visibility-BUISNESSUNIT_AND_MODERATORS=同じグループのすべてのユーザーとモデレーター。
 Visibility-EVERYONE=システムにログインしているすべてのユーザー。
 Visibility-ME_AND_MODERATORS=作成者とモデレーターが表示できます。
 Visibility-PRIVATE=作成者（およびすべての表示レベルに適用される管理者）のみが表示できます。
-visual.sourcesafe=Visual SourceSafe
+visual.sourcesafe=ソースの安全性可視
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

File: frontend/sw360-portlet/src/main/resources/content/Language_ja.properties [hunk 35]
```diff
@@ -1437,23 +1437,23 @@ you.do.not.have.write.access.to.the.project= <b>WRITE</b> 書き込み権限無
...
-ObligationLevel-COMPONENT_OBLIGATION=Component obligations are obligations for a specific component or release only. &#10For example, special measure or actions to be carried out could result from trade compliance &#10or IP issues with the component.
-ObligationLevel-LICENSE_OBLIGATION=License obligation are task to be carried out or risks to be considered from the use of &#10software under a particular license.
-ObligationLevel-PROJECT_OBLIGATION=Project obligations are specific to the projects or products nature and are also requires steps or tasks &#10to be made sure before conveying the software. &#10An example could be tiny hardware with limited printed documentation. In this case open source license &#10information would required special handling, &#10for example print instructions how to obtain OSS license information on the packaging.
+license.obligation=ライセンスオブリゲーション
+obligation.title=オブリゲーションタイトル
+obligation.text=オブリゲーションテキスト
+linked.obligations=リンクされたオブリゲーション
+learn.more.about.obligation.level=よりオブリゲーション詳細について
```
[the remark this answers is not in the document]
Comment (KoukiHama) [reply frontend/sw360-portlet/src/main/resources/content/Language_ja.properties]: I updated it with your advice.

**Comment by mcjaeger** [comment]
general test seem to work: 

Screenshot 2021-08-31 at 23 43 17

Screenshot 2021-08-31 at 23 43 57

Screenshot 2021-08-31 at 23 44 16

Screenshot 2021-08-31 at 23 44 27

Screenshot 2021-08-31 at 23 44 39

**Pull request merged**